cloud_AWS/terraform/module/examples/multiple-vpc-regions/README.md

```diff
@@ -0,0 +1,75 @@
+# Multiple VPC's from different regions
+
+The configuration in this directory creates a configuration for each VPC ID located in the input_data.json file.
+
+
+## Requirements
+
+- Installed and configured AWS CLI
+    - [installation](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+    - [configuration](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html)
+- Created VPC
+- Exported Kentik API credentials:
+
+  ```shell
+  export KTAPI_AUTH_EMAIL="joe.doe@email.com"
+  export KTAPI_AUTH_TOKEN="token123"
+  ```
+
+## Usage
```

Comment:
Please also include the dry-run example with "plan" action, so the reader is aware such an action exists

Reply:
Implemented suggestion